contracts/DataFeedConsumer.sol

```diff
@@ -1,24 +1,27 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.16;
 
-import { IAggregator } from "@bisonai/orakl-contracts/src/v0.1/interfaces/IAggregator.sol";
+import {IAggregator} from "@bisonai/orakl-contracts/src/v0.1/interfaces/IAggregator.sol";
+import {IAggregatorRouter} from "@bisonai/orakl-contracts/src/v0.1/interfaces/IAggregatorRouter.sol";
 
 contract DataFeedConsumer {
     IAggregator internal dataFeed;
+    IAggregatorRouter internal router;
     int256 public answer;
     uint80 public roundId;
 
-    constructor(address aggregatorProxy) {
+    constructor(address aggregatorProxy, address aggregatorRouter) {
```

Comment:
I think passing `AggregatorProxy` loses its purpose after having access to the `AggregatorRouter`. How having two consumer contracts? One showing how to use `AggregatorProxy` and one for `AggregatorRouter`?

Reply:
I felt bothered to make another contract and make separate deploy script, and thought that this is enough explanations for other developers to understand. But I'll write another contract if you insist